README.md

```diff
@@ -1,70 +1,52 @@
+>NOTE: If you need to ensure your code is ready for production use one of the stable libraries.
 
-You found something you'd like to change? Great! Please submit a pull request and we'll do our best to work with you to get your code included into the project.
 
-1. Commit your changes (`git commit -am 'Add some feature'`)
-2. Push to the branch (`git push origin my-new-feature`)
-3. Create new Pull Request
+### Client: Stable
+Last stable versions of packages that have been provided for usage with Azure and are production-ready. These libraries provide you with similar functionalities to the Preview ones as they allow you to use and consume existing resources and interact with them, for example: upload a blob.
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+### Management
+Libraries which enable you to provision specific resources. They are responsible for directly mirroring and consuming Azure service's REST endpoints. You can recognize these libraries by `mgmt` or `arm` in their package names.
 
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+## Need help?
+* For detailed documentation visit our [Azure SDK for JavaScript documentation](https://aka.ms/js-docs)
+* File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-js/issues)
+* Check [previous questions](https://stackoverflow.com/questions/tagged/azure-sdk-js) or ask new ones on StackOverflow using `azure-sdk-js` tag.
 
-If you would like to become an active contributor to this project please follow the instructions provided in [Microsoft Azure Projects Contribution Guidelines](http://azure.github.io/guidelines/).
-
-If you encounter any bugs with the library please file an issue in the [Issues](https://github.com/Azure/azure-sdk-for-js/issues) section of the project.
-
-For additional information about contributing to this project, as well as a getting started guide for contributors, see the [contributing guidelines](CONTRIBUTING.md).
+## Contributing
+For details on contributing to this repository, see the [contributing guide](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md).
 
 ![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js%2FREADME.png)
-
-## Client Library Tested Operating Systems and Node Versions
-
-Currently, the tests for client libraries in this repository are running against:
-
-|             | Linux (Ubuntu 16.04) | MacOS 10.13 | Windows Server 2016 |
-| ----------- | -------------------- | ----------- | ------------------- |
-| **Node 8**  | x                    | x           | x                   |
-| **Node 10** | x                    | x           | x                   |
-| **Node 12** | x                    | x           | x                   |
```

Comment:
Why aren't we listing the Node versions that we support anymore?

Reply:
The table seems like overkill and doesn't include browser information. Further, supported node versions differ from package-to-package as some teams have reasons to support e.g. Node 6 or IE11. So rather than try to enumerate which packages support which platforms, or try to list a "min bar", I thought it better to avoid saying anything in the top-level readme.

Reply:
Then may be we should add a line (not table) on the Node versions supported on the individual readmes ?

Reply:
Fully agreed, language requirements should be listed in individual README prerequisites

Reply:
@ramya-rao-a table seems overkill when you can just say `Node 8 and above`. But yes, each readme should say this, along with any browser versions they've tested with.